manifests/action_promote_author_standby_to_primary.pp

```diff
@@ -20,6 +20,9 @@
   } -> exec { 'set-component.sh author-primary':
     cwd  => $tmp_dir,
     path => ["${base_dir}/aws-tools", '/usr/bin', '/opt/puppetlabs/bin/', '/bin'],
+  } -> exec { 'promote-author-standby-to-primary.sh':
```

Comment:
Can the call to `aws-tools/set-component.sh` be moved to `aws-tools/promote-author-standby-to-primary.sh` ?
The promotion event payload should then execute `aws-tools/promote-author-standby-to-primary.sh`, which will then call `aem-tools/promote-author-standby-to-primary.sh` .
So the idea here is that we have to reverse the call, instead of aem tooling calling aws tooling, it should be aws tooling calling aem tooling. That way the AEM layer doesn't know about the cloud platform.
It's ok for the cloud platform to know about AEM.

Reply:
Done with PR https://github.com/shinesolutions/aem-aws-stack-provisioner/pull/68 and updates this current PR.

Reply:
And PR https://github.com/shinesolutions/aem-aws-stack-builder/compare/master...mbloch1986:enhancement/rename-promoted-author-instance?expand=1